crates/bevy_app/src/schedule_runner.rs

```diff
@@ -48,7 +48,7 @@ impl ScheduleRunnerSettings {
 /// Configures an App to run its [Schedule](bevy_ecs::schedule::Schedule) according to a given
 /// [RunMode]
```

Comment:
We'd be remiss not to update this documentation here, since the plugin no longer owns the `RunMode`. This should be something like

```suggestion
/// Configures an App to run its [`Schedule`](bevy_ecs::schedule::Schedule) according to the [`ScheduleRunnerSettings`] resource.
/// By default, this loops infinitely with no delay (i.e. runs the simulation* as fast as possible)
/// ```rust
/// # use bevy_app::{App, EventWriter, AppExit};
/// let mut counter: u32 = 0;
/// App::new().add_resource(ScheduleRunnerSettings::default())
/// .add_plugin(ScheduleRunnerPlugin).add_system(
/// |exit: EventWriter<AppExit>| if counter >= 10 {exit.send(AppExit)}else{ counter+=1}
);
/// assert_eq!(counter, 10);
/// ```
/// Please note that this plugin does not use windowing**, for that use the winit plugin.
/// 
/// If no runner is added, then the app will simulate once then exit.
```
\* If someone has a suggestion for a word other than simulation, I'm keen to hear it - I feel like it's not quite right here.
\** Better phrasing here would be good - I'm trying to clarify that this will overwrite the winit plugin (if added later).